core/src/test/java/apoc/schema/SchemasEnterpriseFeaturesTest.java

```diff
@@ -105,6 +110,137 @@ public void testKeptNodeKeyAndUniqueConstraintIfExists() {
         });
     }
 
+
+    @Test
+    public void testRelKeyConstraintIsKeptAndDropped() {
```

Comment:
It's hard to understand the test coverage here. I see that:
- uniqueness constraints are in SchemaTest.java.
- relationship indexes I haven't seen even thought there is a parameter for apoc.schema.assert for them. But maybe there are in a different Test as well?
It might be a bit out of scope for this, but I think it would be nice to structure the tests better and maybe rename the Test file to understand what they contain.

Reply:
Uniqueness constraints are community edition, so they don't need docker containers to run, hence the separation :)

This PR is about adding relationship constraints, not indexes, if we are missing index tests maybe that can be a different PR.

SchemaTest --> community available schema proc functionality 
SchemasEnterpriseFeaturesTest --> enterprise available schema proc functionality 

A lot of APOC testing is split into files like this, I think it would be a different task to clean that up!